src/page-modules/departures/departures.module.css

```diff
@@ -33,6 +32,18 @@
   margin-bottom: var(--spacings-medium);
 }
 
+.geolocationButton {
+  height: var(--height);
+  background-color: var(--static-background-background_0-background);
+  border: none;
+  padding: var(--spacings-small);
+  border-bottom-right-radius: 0.75rem;
```

Comment:
I know you just moved this code, but isn't one of the spacings 0.75rem?

Reply:
It does for spacings, but not border-radius 🤔

Reply:
Right! That's why I hadn't used a variable in the first place.